ArWikiCats/translations_formats/classes.py

```
@@ -69,3 +116,15 @@ class YearCountryDataConfig:
+
+
+__all__ = [
+    "TranslationPair",
+    "FormatBotConfig",
+    "CountryBotConfig",
+    "GenreBotConfig",
+    "YearDataConfig",
+    "MultiDataFormatterSecondElementConfig",
+    "MultiDataFormatterConfig",
+    "YearCountryDataConfig",
+]
```

Ruff (0.15.6)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)